### PR TITLE
Cicd pipeline

### PR DESCRIPTION
**Pull request by Chufle**
I added the file (and folders) .github/workflows/deploy.yml to have a cicd pipeline.

### CODE REVIEW

File: infrastructure/dynamodbkutter.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```
Comment (fabianschmauder) [inline infrastructure/dynamodbkutter.tf:1]: basic-dynamodb-table

does not say what it will save

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+      - uses: actions/checkout@v3
+      - name: run terraform
+        run:  |
+          cd infrastructure
+          sh setup_terraform_bucket.sh
+          sh setup_photos_bucket.sh
+          terraform init
+          terraform apply -auto-approve
+
```
Comment (fabianschmauder) [inline .github/workflows/deploy.yml:25]: empty line

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      AWS_REGION: us-west-2
+    steps:
+      - uses: actions/checkout@v3
+      - name: run terraform
+        run:  |
+          cd infrastructure
+          sh setup_terraform_bucket.sh
+          sh setup_photos_bucket.sh
+          terraform init
+          terraform apply -auto-approve
```
Comment (fabianschmauder) [inline .github/workflows/deploy.yml:24]: you could also split it in two steps

- setup s3 buckets

- run terraform

But is not nessesary

**Review by fabianschmauder — COMMENTED** [review]
just small things

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (Chufle) [reply .github/workflows/deploy.yml:25]: I thought at the end of a file there should be an empty line? I am confused...

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (Chufle) [reply .github/workflows/deploy.yml:24]: good idea. I will change it.

File: infrastructure/dynamodbkutter.tf (hunk 1 above)
Comment (Chufle) [reply infrastructure/dynamodbkutter.tf:1]: it is already changed in an earlier pull request of another branch

**Pull request merged**